src/data_store/store.clj

```diff
@@ -0,0 +1,12 @@
+(ns data-store.store
+  (:require
+   [clojure.core.async :refer [<! go-loop timeout]]
+   [data-store.expiration :refer [expire-n]]))
+
+(def store (atom {}))
+(defn dynamic-expiry! [t]
+  (go-loop []
+    (<! (timeout t))
+    (swap! store (fn [previous-store]
+                   (expire-n previous-store (System/currentTimeMillis) 20)))
+    (recur)))
```

Comment:
`swap!` sets `store` to a new value every `t` milliseconds.

`store` is the global data store that the server uses.

It calls `expire-n` to continuously expire 20 keys at a time, as long as more than 25% are expired.